### ISSUE TITLE
a11y: Collapse recipient address on loss of focus

### ISSUE THREAD

**Issue by tshfx** [MetaMask/metamask-extension#6862]
**Describe the bug**
When tabbing through the inputs up a send eth screen, the Recipient Address autocomplete does not collapse, preventing a user from accessing the rest of the form.

**To Reproduce**
1. Go to Send Eth
2. Tab focus (or click on) Recipient Address
3. tab away from the input
4. The autocomplete remains expanded, and the user is unable to see other input fields.

**Expected behavior**
Autocomplete should collapse when it looses focus.

**Screenshots**
![Screen Shot 2019-07-15 at 9 37 12 PM](https://user-images.githubusercontent.com/52763493/61257780-e2703880-a748-11e9-8a86-8f2ce25b336b.png)

**Browser details (please complete the following information):**
- OS: OS X
- Browser: chrome
- MetaMask Version: 6.7.2

**Additional context**
A11y refers to accessibility. This change would improve the extensions screen reader support.

**Comment by Gudahtt** [comment]
I don't think this applies anymore; there is no longer a "Recipient Address" field in our send flow. Instead the choosing of the recipient happens on a separate page of the flow.
